index.html

```diff
@@ -7,5 +7,6 @@
 </head>
 <body>
 Hi Haidee!
+  Hi from Maylis
```

Comment:
Im gona ask you to add a image to the repository and have it linked it here

Reply:
[https://www.rawpixel.com/search/horse?page=1&path=_topics&sort=curated](url)